<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerDialog</class>
 <widget class="QDockWidget" name="LayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer Dialog</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0,0">
    <item>
     <widget class="igQtModelTreeWidget" name="modelTreeWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="igQtModelInformationWidget" name="ModelInformationWidget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>igQtModelInformationWidget</class>
   <extends>QWidget</extends>
   <header>IQWidgets/igQtModelInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>igQtModelTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">IQComponents/igQtmodelTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
